src/components/alerting.tsx

```diff
@@ -1618,6 +1618,26 @@ const AlertTableRow_: React.FC<AlertTableRowProps> = ({ history, obj }) => {
     );
   }
 
+  const getExtensionDropdownItems = (actions: Action[]) => {
```

Comment:
```suggestion
  const getDropdownItemsWithExtension = (actions: Action[]) => {
```

Since we're actually returning all of the dropdown items.

Alternatively, could just return `extensionDropdownItems` and do the `concat` outside this function.

Reply:
That makes sense. Updated the function name to `getDropdownItemsWithExtension`.